clang/test/BoundsSafety/CodeGen/arithmetic-ops-in-counted-by-calls.c

```diff
@@ -1,6 +1,7 @@
+// REQUIRES: system-darwin
 // NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --replace-value-regex "!annotation ![0-9]+" "!tbaa ![0-9]+" "!tbaa\.struct ![0-9]+" "!nosanitize ![0-9]+" "!srcloc ![0-9]+" --prefix-filecheck-ir-name TMP_
-// RUN: %clang_cc1 -O0  -fbounds-safety -triple arm64e-apple-iphoneos -emit-llvm %s -o - | FileCheck %s
-// RUN: %clang_cc1 -O0  -fbounds-safety -triple arm64e-apple-iphoneos -x objective-c -fbounds-attributes-objc-experimental -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -O0  -fbounds-safety -triple arm64e-apple-ios -emit-llvm %s -o - | FileCheck %s
```

Comment:
Thanks for fixing these. We probably need to investigate using `utils/update_cc_test_checks.py` on tests like this because last time I fixed the target triples the script stopped working (it would remove all the`CHECK` lines)